src/Libraries/Tesellation/Tessellation.csproj

```diff
@@ -15,8 +15,8 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.18.0.1400" />
-    <PackageReference Include="MIConvexHull" version="1.1.17.0411" CopyPDB="true" />
-	<PackageReference Include="StarMath" version="2.0.14.1114" CopyPDB="true" />
+    <PackageReference Include="MIConvexHull" version="1.1.17.411" CopyPDB="true" />
+	<PackageReference Include="StarMath" version="2.0.17.1019" CopyPDB="true" />
```

Comment:
any license changes? Have you updated version info in the about box, license txt and third party wiki?

Reply:
Done